pinot-query-planner/src/main/java/org/apache/pinot/query/type/TypeFactory.java

```diff
@@ -70,6 +70,8 @@ private RelDataType toRelDataType(FieldSpec fieldSpec) {
         return createSqlType(SqlTypeName.VARCHAR);
       case BYTES:
         return createSqlType(SqlTypeName.VARBINARY);
+      case BIG_DECIMAL:
```

Comment:
If I remember the code correctly, this BIG_DECIMAL is not passed down correctly broker to server in the thrift protocol. This is also one of the reason I feel we should do integration test at broker level. 
Can you help verify this?

Reply:
Actually, I think the code I read before is for literal type. maybe it is working for the sql type. Let me double check and get back to you